### PR TITLE
Increase covering coverage

### PR DESCRIPTION
**Pull request by Mjh9122**
Increases coverage to reach final conditional

### CODE REVIEW

File: networkx/algorithms/tests/test_covering.py [hunk 1]
```diff
@@ -1,3 +1,5 @@
+from pytest import raises
```
Comment (rossbar) [inline networkx/algorithms/tests/test_covering.py:1]: Just a nit, but the general best-practice is to simply import `pytest` and access the functionality from the `pytest` namespace, for example `pytest.raises`. One reason for doing so is that `raises` looks an awful lot like the Python builtin `raise`, which can easily confuse a reader who is skimming the code.

File: networkx/algorithms/tests/test_covering.py [hunk 2]
```diff
@@ -13,6 +15,15 @@ def test_graph_with_loop(self):
...
         assert nx.min_edge_cover(G) == {(0, 0)}
 
+    def test_graph_with_isolated_v(self):
+        with raises(
+            nx.NetworkXException,
+            match="Graph has a node with no edge incident on it, so no edge cover exists.",
+        ):
+            G = nx.Graph()
+            G.add_node(1)
+            nx.min_edge_cover(G)
```
Comment (rossbar) [inline networkx/algorithms/tests/test_covering.py:25]: Nice work, this should indeed boost the coverage. Just one quick suggestion: it's generally best to only include the minimum amount of code in the context when testing that exceptions are raised. The more specific you are in the test design, the easier it is for someone else to chase down what exactly went wrong if/when the test fails in the future:

```suggestion
        G = nx.Graph()
        G.add_node(1)
        with pytest.raises(
            nx.NetworkXException,
            match="Graph has a node with no edge incident on it, so no edge cover exists.",
        ):
            nx.min_edge_cover(G)
```

**Review by rossbar — COMMENTED** [review]
Thanks for this @Mjh9122 , just a couple quick suggestions.

Also, it might be worthwhile to add another test case for a graph that contains at least one edge. The reason being that a graph without edges is itself often a special case. In this particular instance, the code is straightforward so we know that there should be no difference, but one of the goals of testing is to define what the current behavior is so that if someone changes the implementation in the future, the test suite will fail if the behavior changes.

It's not necessary for this PR, just a thought. If you're interested you might want to check out [pytest's test parametrization feature](https://docs.pytest.org/en/6.2.x/parametrize.html#pytest-mark-parametrize-parametrizing-test-functions) for inspiration on how you could easily expand this core test to multiple cases.

File: networkx/algorithms/tests/test_covering.py (hunk 1 above)
Comment (Mjh9122) [reply networkx/algorithms/tests/test_covering.py:1]: Sounds good, I'll do that in the future

**Review by rossbar — APPROVED** [review]
Thanks for the updates @Mjh9122 , LGTM!

**Review by MridulS — APPROVED** [review]
thanks @Mjh9122!

**Pull request merged**